frontend/src/metabase/components/MetadataInfo/Popover/Popover.tsx

```diff
@@ -0,0 +1,74 @@
+import type { ReactNode } from "react";
```

Comment:
Sometimes scrollbar is redundantly shown in the popover (Chrome / Ubuntu):

https://github.com/metabase/metabase/assets/6830683/a43ed005-cc23-4bc4-b359-7f1f6c6fec92

Reply:
Good catch. Fixed!